nginx/nginx.conf

```diff
@@ -0,0 +1,49 @@
+daemon off;
+worker_processes 1;
+error_log stderr warn;
+pid /tmp/nginx.pid;
+
+events {
+    worker_connections 1024;
+}
+
+http {
+    sendfile on;
+
+    upstream fastapi {
+        server localhost:8080;
+    }
+
+    map $http_authorization $bearer_token {
+        "~^Bearer\s+(.+)$" $1;
+        default "";
+    }
+
+    proxy_cache_path /var/cache/nginx levels=1:2 keys_zone=fastapi_cache:10m max_size=1g inactive=60m;
+
+    server {
+        listen 80;
+        server_name localhost;
+
+        location / {
+            proxy_pass http://fastapi;
+            proxy_set_header Host $host;
+            proxy_set_header X-Real-IP $remote_addr;
+            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+            proxy_set_header X-Forwarded-Proto $scheme;
+
+            proxy_cache fastapi_cache;
+
+            proxy_cache_key "$scheme$proxy_host$request_uri$bearer_token";
+
+            proxy_cache_valid 200 60m;
```

Comment:
🤔 thinking if we can have a better cache lifetime, my guess is that all traces/morphos, simulations, are long live resources (and not open to update much) 
so may be we can increase it (may be i missing smthg so 1h is also might be a good value)

Reply:
yeah, agreed; it's basically a trade off in space.
As it stands, since it's tied to the lifetime of the token, it doesn't really matter.
When the next patch comes, then it will be more important, so keep it in mind.

However, I think doing a client side `Cache Control` would make sense, also; since the renders last "forever", they don't even need to be pulled across the intertubes, IMO.